### DISCUSSION TITLE
Announcing .NET SDK `v0.2.2`, introducing `OpenFgaClient` with `BatchCheck`, `ListRelations` and more

### DISCUSSION THREAD

**Discussion by rhamzeh** [openfga/community#138] [Announcements]
We've just released [`v0.2.2`](https://github.com/openfga/dotnet-sdk/releases/tag/v0.2.2) of the [.NET SDK](https://www.nuget.org/packages/OpenFga.Sdk), this follows similar changes in `v0.2.4` of our JS SDK and is a planned change for the Go and Python SDKs.

The highlights of this release include:
- `SchemaVersion` is now required in `AuthorizationModel`
- `OpenFgaClient` wrapping around `OpenFgaApi` to simplify some interfaces, and introduces client-side:
  - [`BatchCheck`](https://github.com/openfga/dotnet-sdk/tree/main#batch-check)
  - [`ListRelations`](https://github.com/openfga/dotnet-sdk/tree/main#list-relations)
  - [non-transactional `Writes`](https://github.com/openfga/dotnet-sdk/tree/main#non-transaction-mode)
- Fixes and improvements retries and error handling

Take a look at the following links for an overview of the changes:
- https://github.com/openfga/dotnet-sdk#initializing-the-api-client
- https://github.com/openfga/dotnet-sdk#calling-the-api

We're planning for the next release of the SDK to come with a few breaking changes to `OpenFgaApi`, mainly towards how we are dealing with StoreId.

You can still use the existing `OpenFgaApi` class, and documentation for it can be found [here](https://github.com/openfga/dotnet-sdk/tree/v0.2.1#initializing-the-api-client).

We also hope this will make contributing new SDKs easier for the community.

We'd love your feedback on this direction!

**Comment by xXAvoraXx** [comment]
I got this error after v2.2 update -> 'an item with the same key has already been added. key: authorization'
